mavis/tools/arriba.py

```diff
@@ -0,0 +1,39 @@
+from ..constants import COLUMNS, ORIENT, STRAND
+
+from .constants import TRACKING_COLUMN, SUPPORTED_TOOL
+
+
+def get_orient(string):
+    if string == "downstream":
+        return ORIENT.LEFT
+    elif string == "upstream":
+        return ORIENT.RIGHT
+    return ORIENT.NS
+
+
+def convert_row(row):
+    """
+    transforms the aribba output into the common format for expansion. Maps the input column
+    names to column names which MAVIS can read
+    """
+    std_row = {}
+
+    try:
+        std_row[COLUMNS.break1_chromosome], b1_start = row["breakpoint1"].split(":")
+        std_row[COLUMNS.break2_chromosome], b2_start = row["breakpoint2"].split(":")
+
+        std_row[COLUMNS.break1_strand] = row["strand1(gene/fusion)"].split("/")[1]
+        std_row[COLUMNS.break2_strand] = row["strand2(gene/fusion)"].split("/")[1]
+        std_row[COLUMNS.event_type] = row["type"].split("/")[0]
```

Comment:
what do we expect a slash in? (all the tests have a single type). Could you put some examples?

Reply:
Sure I'll add some examples to the tests